### PR TITLE
kalibrate-rtl (new formula)

### PR DESCRIPTION
**Pull request by labeneator**
### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

_You can erase any parts of this template not applicable to your Pull Request._

### New Formulae Submissions:

- [ ] Does your submission pass
`brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)?
Yes
 - No applicable tests
 - Versioning: How do you version git-builds?

- [ ] Have you built your formula locally prior to submission with `brew install <formula>`?
Yes

### CODE REVIEW

File: Library/Formula/kalibrate-rtl.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+class KalibrateRtl < Formula
+  desc "Tool to Kalibrate RTL devices"
+  homepage "https://github.com/steve-m/kalibrate-rtl"
+  url "https://github.com/steve-m/kalibrate-rtl.git", :revision => "aae11c8a8dc79692a94ccfee39ba01e8c8c05d38"
```
Comment (DomT4) [inline Library/Formula/kalibrate-rtl.rb]: You'll need a stable, tagged upstream release here. We won't accept this without one.

File: Library/Formula/kalibrate-rtl.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "fftw"
+  depends_on "librtlsdr"
+
+  patch :DATA
```
Comment (DomT4) [inline Library/Formula/kalibrate-rtl.rb]: Has this been submitted upstream?

Comment (labeneator) [inline Library/Formula/kalibrate-rtl.rb]: Yup: https://github.com/viraptor/kalibrate-rtl/pull/1

File: Library/Formula/kalibrate-rtl.rb (hunk 1 above)
Comment (labeneator) [inline Library/Formula/kalibrate-rtl.rb]: https://github.com/steve-m/kalibrate-rtl/issues seems to be abandoned. Asked https://github.com/viraptor/kalibrate-rtl/pull/1 for  a release tag.

Comment (mikemcquaid) [inline Library/Formula/kalibrate-rtl.rb]: We'll need upstream to have some sort of acknowledgement before we can merge this.

Comment (labeneator) [inline Library/Formula/kalibrate-rtl.rb]: https://github.com/viraptor/kalibrate-rtl/pull/1 merged. Waiting for upstream merge

Comment (DomT4) [inline Library/Formula/kalibrate-rtl.rb]: It seems like that's not an official upstream, or affiliated with the official upstream at all?

**Comment by UniqMartin** [comment]
Closing for a lack of response/information; happy to reopen once there's some activity here.

**Pull request closed without merging**